### PR TITLE
Create initial version of db/seeds.rb

### PR DESCRIPTION
**Pull request by novotnyjakub**
### Context

We need to be able to prime each environment with fake data set to aid testing. We decided that for Alpha we'll be using the seeds approach and later develop a csv import mechanism.

### Changes proposed in this pull request

* add Faker gem
* create seeds file with a few fixed and few random records

### Guidance to review

```bash
rails db:seed:replant # this will truncate the database and seed from the file
```

### CODE REVIEW

**Review by goodviber — APPROVED** [review]
LGTM

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,104 @@
...
+    note: Faker::Lorem.paragraph,
+    created_at: i.days.ago,
+    updated_at: i.days.ago
+  }
+end
+
+# data insertion for the above data
+
+foster_parents.each do |fp|
+  user = User.create!(
```
Comment (goodviber) [inline db/seeds.rb:68]: could you use create_or_find_by ?

Comment (novotnyjakub) [reply db/seeds.rb:68]: We could do that, but I think this seed is really going to be run on a truncated database only. Particularly if we are going to be using Faker on some parts of it.

Comment (novotnyjakub) [reply db/seeds.rb:68]: Btw. feel free to change and adapt this as we go, I think that's the main advantage of having this checked in with the rest of the code. Each PR can make changes here which are adequate for testing..

**Pull request merged**